### PR TITLE
Update Translations 11/20

### PR DESCRIPTION
**Pull request by mattThousand**

### CODE REVIEW

File: android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,5 +6,4 @@
   <string name="exposure_notification_channel_name">Exposure Notification</string>
   <string name="exposure_notification_message">Someone you were near has tested positive for COVID-19. Tap for more info.</string>
   <string name="exposure_notification_title">Possible COVID-19 exposure</string>
-  <string name="background_worker_notification_title">Checking exposures…</string>
```
Comment (artgibson) [inline android/app/src/main/res/values/strings.xml]: @mattThousand @juancruzgs this string keeps getting removed during ./src/locales/pull.sh runs, but is referenced in the code: https://github.com/Path-Check/gaen-mobile/blob/bcc8b881df4a43651694d2ffb2e1d4ff053ef8ba/android/app/src/main/java/org/pathcheck/covidsafepaths/exposurenotifications/common/NotificationHelper.java#L66

Comment (mattThousand) [reply android/app/src/main/res/values/strings.xml]: good catch @artgibson, will fix

**Comment by Ferrumofomega** [comment]
LGTM

**Review by Ferrumofomega — APPROVED** [review]
LTGM

**Pull request merged**